pom.xml

```diff
@@ -18,7 +18,7 @@
 
   <properties>
     <java.version>1.7</java.version>
-    <spark.version>1.5.2</spark.version>
+    <spark.version>1.6.1</spark.version>
```

Comment:
And this will be nit picky, but can you make that commit _before_ the logging class changes?